app/assets/javascripts/index.js

```diff
@@ -1,5 +1,9 @@
 module.exports.Result = require('./Result.js').Result;
 module.exports.ResultSchema = require('./Result.js').ResultSchema;
+module.exports.Measure = require('./Measure.js').Measure;
+module.exports.MeasureSchema = require('./Measure.js').MeasureSchema;
+module.exports.MeasurePackage = require('./MeasurePackage.js').MeasurePackage;
+module.exports.MeasurePackageSchema = require('./MeasurePackage.js').MeasurePackageSchema;
```

Comment:
Are we also adding the ValueSet model and schema to the exports?

Reply:
Yes, but I only just got the Valueset stuff up and running with Concepts/etc.

Reply:
They should be there now (as well as in the template file that builds `index.js`)